CHANGELOG.md

```diff
@@ -46,6 +65,8 @@ has not changed. This is to prevent errors when concurrent writes are made to th
 - Datasource UUIDs are no longer stored in the storage class and are now only stored in the metadata store - [PR #752](https://github.com/openghg/openghg/pull/752)
 - Support dropped for Python 3.8 - [PR #818](https://github.com/openghg/openghg/pull/818). OpenGHG now supports Python >= 3.9.
 
+- Updated internal naming and input data_type to use "flux" rather than "emissions" consistently. - [PR #827](https://github.com/openghg/openghg/pull/827)
```

Comment:
Looks like this should be moved to under the 0.8.0 section as well?

Reply:
This might also be included as a reason this is a breaking change.

Reply:
Ah yeah good catch.